README.md

```diff
@@ -18,33 +20,48 @@ git clone https://github.com/RioChndr/jaksel-language.git
 
 ### Run
 
-If you install from npm, run jaksel script using `jaksel` command
+If you install from npm, run jaksel script using `jaksel -r <filename...>` command
+
+```bash
+jaksel -r example/example1.jaksel
+```
+
+Or run from this repository
+
+```
+node jaksel-interpreter.js -r example/example1.jaksel
+```
+
+### Compile
+
+If you install from npm, compile jaksel script using `jaksel -c <filename...>` command
```

Comment:
Please add some information about this flag

Reply:
Okey